lib/rules/consistent-return.js

```diff
@@ -33,6 +33,18 @@ function isUnreachable(segment) {
     return !segment.reachable;
 }
 
+/**
+* Checks whether a given node is a `constructor` method in an ES6 class
+* @param {ASTNode} node A node to check
+* @returns {boolean} `true` if the node is a `constructor` method
+*/
+function isClassConstructor(node) {
+    return node.type === "FunctionExpression" &&
+        node.parent &&
+        node.parent.type === "MethodDefinition" &&
+        node.parent.kind === "constructor";
+}
```

Comment:
Wondering if this should be put in ast-utils as `isES6Constructor` (or `isClassConstructor`?). And maybe they could also export `isConstructor` which tests for both?